include/simple-mpc/friction-compensation.hpp

```diff
@@ -0,0 +1,56 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 2-Clause License
+//
+// Copyright (C) 2025, INRIA
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+#pragma once
+
+#include <pinocchio/fwd.hpp>
+// Include pinocchio first
+#include <Eigen/Dense>
+
+#include "simple-mpc/fwd.hpp"
+#include <pinocchio/multibody/model.hpp>
+
+namespace simple_mpc
+{
+  using namespace pinocchio;
+
+  /**
+   * @brief Class managing the friction compensation for torque
+   *
+   * It applies a compensation term to a torque depending on dry and
+   * viscuous frictions.
+   */
+  class FrictionCompensation
+  {
+  public:
+    /**
+     * @brief Construct a new Friction Compensation object
+     *
+     * @param actuation_size Dimension of torque
+     */
+    FrictionCompensation(const Model & model, const long actuation_size);
```

Comment:
Actuation size can be deduced from the model,we don't need to pass it.
However there is the case of the freeflyer. Maybe a flag `ignore_first_joint` (or `ignore_first_freeflyer` to be even more specific) can be added with a default value (to true?). 

What do you think?

Reply:
By supposing that all models have a universe joint, we can indeed deduce the actuation from the number of joints. Let's implement that.